aspnetcore/diagnostics/asp0028.md

```diff
@@ -17,11 +17,17 @@ uid: diagnostics/asp0028
 
 ## Cause
 
-On the server machine that supports IPv6, listening to `Any`, rather than `IPv6Any` will either not work or be slower than necessary, because of the [underlying System types implementation](https://github.com/dotnet/runtime/issues/82404).
+[IPv6Any](/dotnet/api/system.net.ipaddress.ipv6any) is preferred to [Any](/dotnet/api/system.net.ipaddress.any) because `Any` is less performant than `IPv6Any`. In some cases, `Any` may not work at all. `Any` has performance problems due to the [underlying System types implementation](https://github.com/dotnet/runtime/issues/82404).
```

Comment:
lets also include the info about server machine supporting IPv6 - it is the case where the analyzer should be applied. Its necessary to have this information

Reply:
> lets also include the info about server machine supporting IPv6 - it is the case where the analyzer should be applied. Its necessary to have this information

I'm not sure where you want that and exactly what to write. I've added  ***Servers that support `IPv6` my report the `ASP0028` diagnostic message.*** to the first paragraph.

Can you use the suggestion feature to make a suggestion?